CodeEdit.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -0,0 +1,25 @@
+{
+  "object": {
+    "pins": [
+      {
+        "package": "swift-argument-parser",
+        "repositoryURL": "https://github.com/apple/swift-argument-parser.git",
+        "state": {
+          "branch": null,
+          "revision": "82905286cc3f0fa8adc4674bf49437cab65a8373",
+          "version": "1.1.1"
+        }
+      },
+      {
+        "package": "swift-commands",
+        "repositoryURL": "https://github.com/qiuzhifei/swift-commands",
```

Comment:
Do we really need this library? I think we can easily run terminal commands without it right?

Reply:
Of cause we can. Just for experiment. I'll remove it later and implement a function by ourself.

Reply:
Awesome, thank you!!